test/CMakeLists.txt

```diff
@@ -162,7 +162,7 @@ target_link_libraries(SerializerTest gtest_main absl::flat_hash_map ${CMAKE_THRE
 gtest_discover_tests(SerializerTest)
 
 add_executable(ParametersTest ParametersTest.cpp)
-target_link_libraries(ParametersTest gtest_main absl::flat_hash_map  ${CMAKE_THREAD_LIBS_INIT})
+target_link_libraries(ParametersTest gtest_main absl::flat_hash_map ${CMAKE_THREAD_LIBS_INIT})
```

Comment:
Is this some formatting tool for `CMakeLists.txt` which we probably should make part of our CI pipeline?

Reply:
Unfortunately it's just me noticing those whitespaces. Also I just realized CLion supports formatting CmakeLists.txt files as well